config/tweaks.json

```diff
@@ -3162,7 +3179,7 @@
     "UndoScript": [
       "netsh interface teredo set state default"
     ],
-    "link": "https://christitustech.github.io/winutil/dev/tweaks/Essential-Tweaks/Teredo"
+    "link": "[auto generated if i understand #2481 corectly]"
```

Comment:
You don't need to update it, as from what I understand, it'll be overwritten to the category & name of tweak, see [these lines](https://github.com/ChrisTitusTech/winutil/pull/2481/files#diff-ad8a830e5d482eefc3e300b32b2daa2c950fe09de54e0f66c2e4b86707913b98R534-R563) for more info, most notably is line 553, which's the value of link to use, hope this clears some things up 👍.